src/HtmlUtils.tsx

```diff
@@ -625,7 +625,7 @@ export function bodyToHtml(content: IContent, highlights: Optional<string[]>, op
         "markdown-body": isHtmlMessage && !emojiBody,
     });
     if (opts.emotes) {
-        const tmp = strippedBody?.replace(CUSTOM_EMOTES_REGEX, (m) => (opts.emotes.get(m) ? opts.emotes.get(m) : m));
+        const tmp = strippedBody?.replace(CUSTOM_EMOTES_REGEX, (m) => (opts?.emotes?.get(m) ? opts?.emotes?.get(m)! : m));
```

Comment:
the condition side could/should use `has`

Reply:
done